index.bs

```diff
@@ -320,8 +322,15 @@ Issue: Include cross references to the specs for these hash functions.
     value.  This signals to the listening agent that it should connect to the
     advertising agent to discover updated metadata.
 
-Issue: Add examples of sample mDNS records.
+The advertising agent may add an additional field to the TXT record:
```

Comment:
Perhaps make it a "should" instead of "may" and briefly explain why one should do it.

Reply:
Done.